src/core/toga/app.py

```diff
@@ -106,6 +107,20 @@ def icon(self):
     def icon(self, name):
         self._icon = Icon.load(name, default=self.default_icon)
 
+    @property
+    def main_window(self):
```

Comment:
Setting this on the app definitely feels cleaner.

Reply:
Yeah. I don't know why this didn't occur to me sooner.